src/ontology/mondo-ingest.Makefile

```diff
@@ -351,14 +351,46 @@ deploy-mondo-ingest:
 
 USE_MONDO_RELEASE=false
 
+# TODO: make sure passes aftr add this line back below cd mondo/src/ontology
+#		make mondo.owl mappings -B MIR=false IMP=false MIR=false &&\
+
 tmp/mondo_repo_built:
 	if [ $(USE_MONDO_RELEASE) = true ]; then wget http://purl.obolibrary.org/obo/mondo.owl -O $@; else cd $(TMPDIR) &&\
 		rm -rf ./mondo/ &&\
 		git clone --depth 1 https://github.com/monarch-initiative/mondo &&\
 		cd mondo/src/ontology &&\
 		make mondo.owl mappings -B MIR=false IMP=false MIR=false &&\
+		latest_hash=$$(git rev-parse origin/master) &&\
 		cd ../../../../ &&\
-		touch $@; fi
+		echo "$$latest_hash" > $@; fi
+
+# TODO
+# explain what does here. explain that -B is just in case tmp/mondo_repo_built exists but not the rpeo
+.PHONY: refresh-mondo-clone
+refresh-mondo-clone:
```

Comment:
## Preliminary results
@matentzn @twhetzel So far I got this working. It was a little trickier than I'd hoped; the intersecting of ShellScript and makefile always feels a bit frustrated.

So far these (4) conditions seem to be working:

1. If mondo repo clone does not exist in `tmp/`, makes it
2. If it exists and is up to date, do nothing
3. If it exists, and I've manually changed the hash in `tmp/mondo_repo_built`, it re-creates the mondo clone
4 If it exists, and latest commit hash on `mondo` `origin/master`  is different than what is in `tmp/mondo_repo_built`, it re-creates the mondo clone

My next task is to put `$(MAKE) refresh-mondo-sssom` or `$(MAKE) refresh-mondo-owl` in the bodies of the respective goals that use them, and remove `monso.sssom.tsv` and `mondo.owl` from the prereqs of those goals.

- [ ] @joeflack4 Maybe I should rename the `refresh-` in `refresh-mondo-sssom` etc to `updated-` or `up-to-date`. Because it won't always refresh if the thing is up to date; it's having an up-to-date file that is the idea.

Reply:
The goal name is fine with with me. I take `refresh-` to mean do something to get the latest version available.

Reply:
Fine. Per irrelevant conventions I tended to use:

1. Refresh for imports
2. Recreate for components
3. Update for anything else

This is not very consistent, so you go with what you like